candid/evm_rpc.did

```diff
@@ -36,6 +36,7 @@ type EthMainnetService = variant {
   BlockPi;
   Cloudflare;
   PublicNode;
+  Llama;
```

Comment:
Currently using `Llama` in place of `LlamaNodes` since the API domain name is `llamarpc.com`. Otherwise, also having `PublicNode` could create some confusion about when to include an `s` at the end for people unfamiliar with these individual providers.

Reply:
I thought those were coming from the company name, which in that case `LlamaNodes` and `PublicNode` make perfect sense, and not from the API domain name. A priori I would say that the company name is more stable than the domain name, that's why I would tend to stick to the company name, but that's not a strong opinion.